ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ReportJobs.tsx

```diff
@@ -91,7 +107,7 @@ function ReportJobs({ scanConfig, isComplianceReportingEnabled }: ReportJobsProp
         });
         setSearchFilter({
             ...searchFilter,
-            'Report State': newFilters,
+            'Compliance Report State': newFilters,
```

Comment:
The filters and sorting terms are a bit different with Compliance Reporting. Making changes for those.